### ISSUE TITLE
Heading captions not read out when navigating by heading in a screenreader

### ISSUE THREAD

**Issue by timpaul** [alphagov/govuk-frontend#1505]
When heading captions are marked up in spans (as they are in the Design System itself) they are not read out when a user navigates a page via the headings in a screenreader.

It's been suggested that we update our guidance and examples to recommend that when a caption appears before a H1 is should be marked up as a H2, and when it appears before other headings it should be nested inside them.

**Comment by timpaul** [comment]
The Design System team has reviewed this issue and decided not to action it at this time. 

There are [two ways of marking up headings using the Design System](https://design-system.service.gov.uk/styles/typography/#headings-with-captions); one that reads out the caption and one that doesn't. Therefore the recommended behaviour can be supported if the designer wants to.
